beacon-chain/sync/initial-sync/initial_sync_test.go

```diff
@@ -276,9 +276,7 @@ func connectPeerHavingBlocks(
 	p := p2pt.NewTestP2P(t)
 
 	p.SetStreamHandler("/eth2/beacon_chain/req/beacon_blocks_by_range/1/ssz_snappy", func(stream network.Stream) {
-		defer func() {
-			assert.NoError(t, stream.Close())
-		}()
+		defer stream.Close()
```

Comment:
Please handle this error from stream.Close(). (Static analysis will not allow the build to complete with unhandled errors)